### ISSUE TITLE
BasicConv2d 没有定义，请问他和RFBNet中的定义一致吗？

### ISSUE THREAD

**Issue by qaxuDev** [djzgroup/CFENet#1]
https://github.com/djzgroup/CFENet/blob/19b24e4e17e29799507d47cf27c4d491aa90c8a4/network/model.py#L270

https://github.com/GOATmessi7/RFBNet/blob/master/models/RFB_Net_vgg.py
```
class BasicConv(nn.Module):

    def __init__(self, in_planes, out_planes, kernel_size, stride=1, padding=0, dilation=1, groups=1, relu=True, bn=True, bias=False):
        super(BasicConv, self).__init__()
        self.out_channels = out_planes
        self.conv = nn.Conv2d(in_planes, out_planes, kernel_size=kernel_size, stride=stride, padding=padding, dilation=dilation, groups=groups, bias=bias)
        self.bn = nn.BatchNorm2d(out_planes,eps=1e-5, momentum=0.01, affine=True) if bn else None
        self.relu = nn.ReLU(inplace=True) if relu else None

    def forward(self, x):
        x = self.conv(x)
        if self.bn is not None:
            x = self.bn(x)
        if self.relu is not None:
            x = self.relu(x)
        return x
```

**Comment by djzgroup** [comment]
BasicConv2d consists of  Conv layer, batchnormal layer, and ReLU layer.
